### PR TITLE
Made few changes to sdk-feature-store 

### PR DESCRIPTION
**Pull request by Tharun-SpringML**
Made few changes to sdk-feature-store file
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by ivanmkc** [comment]
Please be more descriptive with PR titles and descriptions.

File: notebooks/official/feature_store/sdk-feature-store.ipynb [hunk 1]
```diff
@@ -0,0 +1,1261 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:1]: <p>This Colab --&gt; This notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/486/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:1]: <p>using the following methods. -&gt; using the <code>create_feature</code> method. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/486/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:1]: <p>no future tense. drop the word 'will'.</p><p> </p><p>no 1st person, use 2nd person. Replace 'we' with 'you'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/486/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:1]: <p>Colab -&gt; notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/486/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:1]: <p>Batch Serving -&gt; Batch serving</p><p> </p><p>no future tense - drop the word 'will'</p><p> </p><p>Title needs to be a verb phrase (currently a noun phrase)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/486/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

**Review by andrewferlitsch — APPROVED** [review]

**Comment by andrewferlitsch** [comment]
This notebook passed the automated test. But the test is running other unrelated notebook which failed:

get_started_vertex_datasets.ipynb                               FAILED    00:03:41    https://console.cloud.google.com/cloud-build/builds/3a3e3994-b00b-47cd-abe9-83ed27faaf6b?project=1012616486416
Step #4: get_started_bqml_training.ipynb                                 FAILED    00:12:48    https://console.cloud.google.com/cloud-build/builds/5ecfc284-3910-4129-a4ed-ee1d8865606d?project=1012616486416
Step #4: get_started_vertex_training.ipynb                               FAILED    00:22:29    https://console.cloud.google.com/cloud-build/builds/798b0183-f61c-499b-93a0-1bbe76aeb59e?project=1012616486416
Step #4: get_started_with_cmek_training.ipynb                            FAILED    00:03:37    https://console.cloud.google.com/cloud-build/builds/2f44cd32-6ea5-4d51-8d60-44e357443f99?project=1012616486416
Step #4: get_started_with_tf_serving.ipynb                               FAILED    00:03:28    https://console.cloud.google.com/cloud-build/builds/6e8ba911-8125-424a-8631-51bbdcbf790a?project=1012616486416

Since it passed, I am manually merging the notebook

**Pull request merged**